docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md

```diff
@@ -0,0 +1,43 @@
+# Serial port connection guide
+
+This document describes how to set up a serial connection between RTE and
+another device.
+
+## Prerequisites
+
+* [Prepared RTE](../v1.1.0/quick-start-guide.md)
+* Two jumpers
+* Choose one of the following cables depending on the connectivity to your
+    platform:
+    - RS232-RJ45 cable
+    - RS232-RS232 cable
+    - 3 wire cables
```

Comment:
In some cases there's an option to use also USB-RS232 converter and 3-wire cable